src/utils/trigger-column-utils.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Label, Truncate, Flex, FlexItem } from '@patternfly/react-core';
+import { CommitIcon } from '~/components/Commits/CommitIcon';
+import { PipelineRunEventType } from '../consts/pipelinerun';
+import { ExternalLink } from '../shared';
+
+export interface TriggerColumnData {
+  gitProvider?: string;
+  repoOrg?: string;
+  repoURL?: string;
+  prNumber?: string;
+  eventType?: string;
+  commitId?: string;
+}
+
+/**
+ * Renders the trigger column data with appropriate icons and links
+ * @param data - Object containing trigger-related data
+ * @returns JSX element for the trigger column
+ */
+export const getTriggerColumnData = (data: TriggerColumnData): React.ReactNode => {
```

Comment:
hmm, I think it would make more sense to have it as a component instead of function, what do you think? :) (e.g. `TriggerColumnData`)